frontend/src/app/utils/TableUtils.jsx

```diff
@@ -69,13 +71,36 @@ function CustomNoRows() {
     );
 }
 
-export default function TableUtils({ dataContent, columns, setRowSelected, path }) {
+export default function TableUtils({ dataContent, columns, setRowSelected, path, type, titleToolTip }) {
+    const LINK_OF_REPORTS_FISHERMEN = "http://localhost:8000/api/reports/fishermen"
```

Comment:
```suggestion
    const LINK_OF_REPORTS_FISHERMEN = "http://localhost:8000/api/reports/fishermen"
```

Mano, o ideal aqui seria usar aquela env REACT_APP_API_BASE_URL